examples/self_managed_node_group/main.tf

```diff
@@ -374,14 +374,24 @@ resource "aws_kms_key" "eks" {
   tags = local.tags
 }
 
-data "aws_ami" "bottlerocket_ami" {
+data "aws_ami" "eks_default" {
+  filter {
```

Comment:
Let's move `filter` block after `owners` as it was.

Reply:
fixed in d147a691f4d5f7a0bba9080976d46a9b61e8a23c